### ISSUE TITLE
installing the plugin for ios just keeps on waiting

### ISSUE THREAD

**Issue by feanor07** [mapsplugin/cordova-plugin-googlemaps#1120]
My development platform is set to IOS and I am running the following command to install the plugin and it never ends:

cordova plugin add cordova-plugin-googlemaps --variable API_KEY_FOR_ANDROID="MY_ANDROID_KEY" --variable API_KEY_FOR_IOS="MY_IOS_KEY"

This is the output written to my console:

Fetching plugin "cordova-plugin-googlemaps" via npm
Installing "plugin.google.maps" for ios
Fetching plugin "https://bitbucket.org/nightstomp/cordova-plugin-googlemaps-sdk.git" via git clone

I tried cordova plugin add cordova-plugin-googlemaps --variable API_KEY_FOR_IOS="MY_IOS_KEY"
and still the same resut.

**Comment by wf9a5m75** [comment]
That's old.

Try to install from the master branch.
```
cordova plugin add https://github.com/mapsplugin/cordova-plugin-googlemaps --variable API_KEY_FOR_ANDROID="MY_ANDROID_KEY" --variable API_KEY_FOR_IOS="MY_IOS_KEY"
```

**Comment by feanor07** [comment]
well I waited for the command to finish a little bit longer (it executes quite fast for android) and it succeeded. However; when I build the app it fails with following warnings and errors:


/Users/macpro-test3/Desktop/coding/cordovamapstrial/cordova/platforms/ios/Cordovamapstrial/Plugins/plugin.google.maps/GoogleMaps.m:24:30: warning: incompatible pointer types assigning to 'UIWebView *' from 'UIView *' [-Wincompatible-pointer-types]
    self.pluginLayer.webView = self.webView;
                             ^ ~~~~~~~~~~~~
/Users/macpro-test3/Desktop/coding/cordovamapstrial/cordova/platforms/ios/Cordovamapstrial/Plugins/plugin.google.maps/GoogleMaps.m:28:45: warning: incompatible pointer types assigning to 'UIWebView *' from 'UIView *' [-Wincompatible-pointer-types]
    self.pluginScrollView.debugView.webView = self.webView;
                                            ^ ~~~~~~~~~~~~
/Users/macpro-test3/Desktop/coding/cordovamapstrial/cordova/platforms/ios/Cordovamapstrial/Plugins/plugin.google.maps/GoogleMaps.m:34:30: warning: incompatible pointer types assigning to 'UIWebView *' from 'UIView *' [-Wincompatible-pointer-types]
    self.pluginLayer.webView = self.webView;
                             ^ ~~~~~~~~~~~~
/Users/macpro-test3/Desktop/coding/cordovamapstrial/cordova/platforms/ios/Cordovamapstrial/Plugins/plugin.google.maps/GoogleMaps.m:167:30: warning: incompatible pointer types assigning to 'UIWebView *' from 'UIView *' [-Wincompatible-pointer-types]
        self.mapCtrl.webView = self.webView;
                             ^ ~~~~~~~~~~~~
/Users/macpro-test3/Desktop/coding/cordovamapstrial/cordova/platforms/ios/Cordovamapstrial/Plugins/plugin.google.maps/GoogleMaps.m:176:59: error: no known instance method for selector 'initWithWebView:'
        Map *mapClass = [[NSClassFromString(@"Map")alloc] initWithWebView:self.webView];
                                                          ^~~~~~~~~~~~~~~
/Users/macpro-test3/Desktop/coding/cordovamapstrial/cordova/platforms/ios/Cordovamapstrial/Plugins/plugin.google.maps/GoogleMaps.m:186:38: warning: incompatible pointer types assigning to 'UIWebView *' from 'UIView *' [-Wincompatible-pointer-types]
            self.pluginLayer.webView = self.webView;
                                     ^ ~~~~~~~~~~~~
/Users/macpro-test3/Desktop/coding/cordovamapstrial/cordova/platforms/ios/Cordovamapstrial/Plugins/plugin.google.maps/GoogleMaps.m:245:68: error: no known instance method for selector 'initWithWebView:'
                pluginClass = [[NSClassFromString(className)alloc] initWithWebView:self.webView];
                                                                   ^~~~~~~~~~~~~~~
/Users/macpro-test3/Desktop/coding/cordovamapstrial/cordova/platforms/ios/Cordovamapstrial/Plugins/plugin.google.maps/GoogleMaps.m:321:19: error: no visible @interface for 'UIView' declares the selector 'stringByEvaluatingJavaScriptFromString:'
    [self.webView stringByEvaluatingJavaScriptFromString:jsString];
     ~~~~~~~~~~~~ ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/Users/macpro-test3/Desktop/coding/cordovamapstrial/cordova/platforms/ios/Cordovamapstrial/Plugins/plugin.google.maps/GoogleMaps.m:404:34: warning: incompatible pointer types assigning to 'UIWebView *' from 'UIView *' [-Wincompatible-pointer-types]
        self.pluginLayer.webView = self.webView;
                                 ^ ~~~~~~~~~~~~

**Comment by feanor07** [comment]
When I try from the master as you suggested; everything works like a charm! This hit me quite bad; I lost couple of hours trying to understand what is going on. This was at a new-setup project and I thought I did the exact same things like before. However; now I've realised I have used master instead of stable build at my previous project and this was the difference why I did not encounter any problems.

Anyway; the issue has turned into getting errors from the stable build; whereas it is working by using the master. Thanks for the quick response by the way.
